src/main/webapp/custom/js/init.js

```diff
@@ -150,6 +151,31 @@
                 $('.btn-group').find('a.underline').addClass('btn-info'):
                     $('.btn-group').find('a.underline').removeClass('btn-info');
             });
+        },
+
+        initSearch: function() {
+
+            $('#search-tag').chosen({
+                width: '100%',
+                no_results_text: 'Add a tag and press enter(min 3 chars)'
+            }).trigger('chosen:updated');
+
+            $('#search-title').chosen({
+                width: '100%',
+                no_results_text: 'Add a tag and press enter(min 3 chars)'
+            }).trigger('chosen:updated');
+
+            $('#search-keyword').chosen({
+                width: '100%',
+                no_results_text: 'Add a tag and press enter(min 3 chars)'
+            }).trigger('chosen:updated');
+
+            $('#search-entity').chosen({
+                width: '100%',
+                no_results_text: 'Add a tag and press enter(min 3 chars)'
+            }).trigger('chosen:updated');
```

Comment:
create private function like, will avoid lots of code
_seachTag(tag, width, text){
   $("#"+tag).chosen({
       width: width
     ,no_result_text: text
   }).trigger("chosen:updated");
}